src/screens/inbox/components/InboxItem.tsx

```diff
@@ -57,6 +60,7 @@ export const InboxItemComponent = (props: InboxItemProps) => {
               numberOfLines={1}
               style={tailwind.style(
                 'text-base font-inter-medium-24 tracking-[0.24px] text-gray-950 capitalize',
+                `max-w-[${width - 250}px]`,
```

Comment:
@muhsin-k Can you verify does this work or not? I guess it won't.

Reply:
@iamsivin 
| Before | After |
| ------------- | ------------- |
| <img width="434" alt="image" src="https://github.com/user-attachments/assets/7102461c-a8a5-42ab-a66a-fcb0383bc243" />  | <img width="434" alt="image" src="https://github.com/user-attachments/assets/c8b519e6-5fbc-423c-b09e-140af541f67b" />  |